homeassistant/components/ozw/__init__.py

```diff
@@ -346,7 +347,8 @@ def async_handle_scene_activated(hass: HomeAssistant, scene_value: OZWValue):
         scene_value_id = scene_value.value["Selected_id"]
 
     _LOGGER.debug(
-        "[SCENE_ACTIVATED] node_id: %s - scene_id: %s - scene_value_id: %s",
+        "[SCENE_ACTIVATED] scene_instance: %s - node_id: %s - scene_id: %s - scene_value_id: %s",
```

Comment:
I'd say that it's not the scene instance but the ozw instance. The scene is an `OZWValue`. The instance is an `OZWInstance`.

Reply:
Yup sorry just caught that, fixing.